compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -190,15 +191,7 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
     }
 }
 
-trait ResolverAstLoweringExt {
-    fn legacy_const_generic_args(&self, expr: &Expr) -> Option<Vec<usize>>;
-    fn get_partial_res(&self, id: NodeId) -> Option<PartialRes>;
-    fn get_import_res(&self, id: NodeId) -> PerNS<Option<Res<NodeId>>>;
-    fn get_label_res(&self, id: NodeId) -> Option<NodeId>;
-    fn get_lifetime_res(&self, id: NodeId) -> Option<LifetimeRes>;
-    fn take_extra_lifetime_params(&mut self, id: NodeId) -> Vec<(Ident, NodeId, LifetimeRes)>;
-}
-
+#[extension]
```

Comment:
@danielhenrymantilla's ext_trait crate uses syntax that puts the tokens `trait ResolverAstEtcEtc` into the source code like this, which makes it easier to search for the definition. I think doing it that way (or just using his crate if you can deal with the worst formatting ever :3) would be useful.

Reply:
done